geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-orc/src/main/scala/org/locationtech/geomesa/fs/storage/orc/utils/OrcAttributeReader.scala

```diff
@@ -359,6 +362,24 @@ object OrcAttributeReader {
     }
   }
 
+  // reads a WKB geometry from a vector and sets it in a simple feature
+  class GeometryWkbReader(val vector: BytesColumnVector, val attribute: Int)
+      extends VectorReaderAdapter[BytesColumnVector] {
+    override def getValue(row: Int): AnyRef = {
+      if (vector.noNulls || !vector.isNull(row)) {
```

Comment:
Is the vector.noNulls cheap?  If not, is it worth factoring out?

Reply:
it's just a boolean var i believe, we set it explicitly on write.

Reply:
it would be tricky to factor out b/c it can change when a new record batch is loaded.